src/services/keyboard_actions.js

```diff
@@ -231,6 +231,66 @@ const DEFAULT_KEYBOARD_ACTIONS = [
         description: "Open new empty window",
         scope: "window"
     },
+    {
+        actionName: "firstTab",
+        defaultShortcuts: ["CommandOrControl+1"],
+        description: "Activates the first tab in the list",
+        scope: "tabs"
```

Comment:
Scope in this context means on which element the keyboard shortcuts are attached - this means that for the shortcut to work, the focus has to be inside the element.

So e.g. shortcuts with "note-tree" scope work only when the focus is in note tree. This allows to have the same shortcut have different actions attached based on the context - e.g. CTRL-C in note tree does something a bit different from CTRL-C in the text editor.

However here I don't see a need for a separate scope - these should be "global" - active on the whole application, which corresponds to the "window" scope.

Reply:
I definitely agree it didn't seem necessary, I just wasn't sure if doing `keyboardActionService.setupActionsForElement('window', $(document), this);` would cause some other issues with other listeners, but after testing it seems fine so I'll switch it back to `window`